### ISSUE TITLE
Broken Special:Upload link in the sidebar

### ISSUE THREAD

**Issue by kelson42** [openzim/mediawiki-docker#1]
The link in the sidebar:
```
https://mediawiki.openzim.org/wiki/Special:Upload
```
fails 

But:
``` 
https://mediawiki.openzim.org/index.php?title=Special:Upload
```
works

There is a problem I guess in the reverse proxy configuration.

A few remarks about that:
* Using Apache is OK but quite overkill for just a reverse proxy. Nginx would be more appropriate, I think this is what was used in WikiFundi 1.0
* To avoid collisions on the `/` level (for example if you want to setup something like `/favicon.png`, Wikipedia has moved the Mediawiki on the `w/` and `wiki/ paths I would recommend to stick to that configuration. I think WikiFundi was also configured that way.
* Doing so and reuse the Nginx configuration like in WikiFundi would avoid all these problem.

**Comment by florentk** [comment]
I have just forget to change the relativ url in the configuration of UploadWizard extension. 
But, I can use Nginx also.

**Comment by florentk** [comment]
fix in commit 121a5aa
